### PR TITLE
Hyundai: add FW for 2018 Elantra GT Sport

### PR DESCRIPTION
**Pull request by neverbread**
<!--- ***** Template: Fingerprint *****

**Car**
2018 Elantra GT Sport

**Route**

3f3b34eca77d1332

### CODE REVIEW

**Comment by sshane** [comment]
What is your dongle id?

Also was wondering if these cars can steer to zero, can you change the minSteerSpeed to 0?

**Comment by neverbread** [comment]
> What is your dongle id?
> 
> Also was wondering if these cars can steer to zero, can you change the minSteerSpeed to 0?

My dongle ID is 3f3b34eca77d1332. I'm rifkers in the discord. Changed minSteerSpeed in my openpilot branch. Still testing it :)

**Comment by sshane** [comment]
@neverbread good to merge? Any issues?

**Comment by neverbread** [comment]
Yep, good to merge! No issues.

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -1777,22 +1777,27 @@ def match_fw_to_car_fuzzy(live_fw_versions) -> Set[str]:
     (Ecu.fwdCamera, 0x7c4, None): [
       b'\xf1\x00PD  LKAS AT USA LHD 1.01 1.01 95740-G3100 A54',
       b'\xf1\x00PD  LKAS AT KOR LHD 1.00 1.02 95740-G3000 A51',
+      b'\xf1\x00PD  LKAS AT USA LHD 1.00 1.02 95740-G3000 A51',
     ],
     (Ecu.transmission, 0x7e1, None): [
       b'\xf1\x006U2V0_C2\x00\x006U2VA051\x00\x00DPD0H16NS0e\x0e\xcd\x8e',
       b'\xf1\x006U2U0_C2\x00\x006U2T0051\x00\x00DPD0D16KS0u\xce\x1fk',
+      b'\xf1\x006U2V0_C2\x00\x006U2V8051\x00\x00DPD0T16NS4\x00\x00\x00\x00',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:1785]:
```suggestion
      b'\xf1\x006U2V0_C2\x00\x006U2V8051\x00\x00DPD0T16NS4\xda\x7f\xd6\xa7',
      b'\xf1\x006U2V0_C2\x00\x006U2V8051\x00\x00DPD0T16NS4\x00\x00\x00\x00',
```

**Pull request merged**